### PR TITLE
Msl 491

### PR DESCRIPTION
**Pull request by wmickeyd**

### CODE REVIEW

**Comment by dpeterka** [comment]
Please rebase (squash) when ready to a single commit
Remove all the keys
Credit or re-write the portions you have borrowed from: https://github.com/Mayeu/ansible-playbook-sensu/blob/master/defaults/main.yml

File: jenkins_dsl/ami.groovy [hunk 1]
```diff
@@ -30,6 +30,12 @@ def amis = [
                   "ami_profile":"jetty8",
                   "branch" : "master"
                 ],
+              "ami-jetty9":
```
Comment (dpeterka) [inline jenkins_dsl/ami.groovy]: I am confused why this isn't sensu role

File: roles/sensu/files/sensu_client_cert.pem [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+VFc9zM3TYCVth0J+qpmjEEMlrT1Yh/2c8yXKtJdxIsD82jHev5aSMkC4COImLI0g
+VTkBvTgrBrFCamHUQ04lhVIyUlBLJ/mfEzYI0WM8I/tnSwZYEnD6gJ+sQzDp2X76
+c7vIIunhIOtDQp56Iz6GvH1Xs1AW3I63BUq5f89grfVdpNsJLwXPeXA2YGvtTamj
+iQIDAQABMA0GCSqGSIb3DQEBBQUAA4IBAQCY/iwMjrcUg8OHujMcLnSbmbuGakNM
+BbT4nN6eRqRNchqx3phFL2oALKQS8k6O1A3wnQPO9bEuhM2bILaU0TLctb+r+31J
+Qmwz+QZF+h5MLDOEXG+kDlcD1HdJTAsYRQz7z0ejsM6+uvng97l0lzjIA6/JYzh3
+eFbevlrKD3NOkzFrmy6k5OiYbP10hWcQ/HQ0QHaZr3D+qgOqoy2CUexMYmgs8v83
+ETFGzmDr7VpB7S1ua4jdnL4m3ut6RHX9vuS/NVGue52Gbt9c4DXmvQO/7AttQTZ1
+4O4vJGwoNwL/TxEJm7qBriudOQe5gYsE9CGu/znT/2iGxdMzNITb9coG
+-----END CERTIFICATE-----
```
Comment (dpeterka) [inline roles/sensu/files/sensu_client_cert.pem]: Never check in a private key

File: roles/sensu/tasks/client.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    mode=0750
+
+- name: generate config files
+  template:
+    src=client.json.j2
+    dest=/etc/sensu/conf.d/client.json
+    owner=sensu
+    group=sensu
+    mode=0640
+    backup=yes
```
Comment (dpeterka) [inline roles/sensu/tasks/client.yml]: Don't need to backup

File: roles/sensu/tasks/common.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  apt_key: url=http://repos.sensuapp.org/apt/pubkey.gpg state=present
+
+- name: add the official Sensu repository
+  copy:
+    src=sensu-server.list
+    dest=/etc/apt/sources.list.d/
+    backup=yes
+  register: aptrepo
+
+- name: refresh apt cache
```
Comment (dpeterka) [inline roles/sensu/tasks/common.yml]: This is old you can just set this when you install a package

File: roles/sensu/tasks/common.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    dest=/etc/sensu/ssl/{{ item }}
+    owner="{{ sensu_user }}"
+    group="{{ sensu_group }}"
+    mode=0640
+    backup=yes
+  with_items:
+    - "{{ sensu_cert_file_name }}"
+    - "{{ sensu_key_file_name }}"
+  when: not sensu_server_rabbitmq_insecure
+  notify:
```
Comment (dpeterka) [inline roles/sensu/tasks/common.yml]: Probably don't need to restart for first boot.

File: roles/sensu/tasks/server.yml [hunk 6]
```diff
@@ -0,0 +1,74 @@
+---
+- name: copy the patched init script
```
Comment (dpeterka) [inline roles/sensu/tasks/server.yml]: What did you need to patch?

File: sensu.yml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+- hosts: sensu
```
Comment (dpeterka) [inline sensu.yml]: This should be using package: not apt:

File: jenkins_dsl/ami.groovy (hunk 1 above)
Comment (wmickeyd) [inline jenkins_dsl/ami.groovy]: That actually was something I missed from the last pull request for jetty 9, I'll remove it from this one

**Pull request closed without merging**